django/test/runner.py

```diff
@@ -625,6 +625,26 @@ def is_discoverable(label):
     return os.path.isdir(os.path.abspath(label))
 
 
+def reorder_postprocess(reordered_suite):
+    # To make the TransactionTestCase initialize their data properly,
+    # they should know if the next TransactionTestCase needs initial data
+    # migrations serialized in the connection.
+    # Initialize _next_serialized_rollback attribute depending on the
+    # serialized_rollback option present in the next test in the test suite.
+    for previous_test, next_test in zip(
+        reordered_suite._tests[:-1],
+        reordered_suite._tests[1:]
+    ):
+        next_serialized_rollback = getattr(next_test, 'serialized_rollback', None)
+
+        # If the next test has no serialized_rollback option, it means that we
+        # have reached the end of the TransactionTestCases.
+        if next_serialized_rollback is not None:
+            previous_test._next_serialized_rollback = next_serialized_rollback
+
```

Comment:
Drop the blank lines in this function - there are really only five lines which isn't hard to read.